maven-xml-impl/pom.xml

```diff
@@ -48,6 +48,12 @@ under the License.
       <artifactId>plexus-xml</artifactId>
       <version>${plexusXmlVersion}</version>
       <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.maven</groupId>
+          <artifactId>maven-xml-impl</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Doesn't the current module take precedence during resolution ?
What kind of issues does it actually cause ?

Reply:
> Doesn't the current module take precedence during resolution ? What kind of issues does it actually cause ?
in  my landtop，windows11 ,apache-maven-3.9.4
run command 'mvn clean compile  -Dmaven.test.skip=true ' in maven home_dir(my case is D:\works\git\maven),it occured an exception as below:
[ERROR] Failed to execute goal on project maven-xml-impl: Could not resolve dependencies for project org.apache.maven:maven-xml-impl:jar:4.0.0-alpha-8-SNAPSHOT: The following artifacts could not be resolved: org.apache.maven:maven-xml-impl:jar:4.0.0-alpha-8-SNAPSHOT (absent): org.apache.maven:maven-xml-impl:jar:4.0.0-alpha-8-SNAPSHOT was not found in https://xxxxxxxxxx/repository/public during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of aliyun has elapsed or updates are forced -> [Help 1]
.......
**It looks like maven-xml-impl is needed to get plexus-xml, but maven-xml-impl is being built**

Reply:
I wonder if that's a bug somehow in the resolution... @cstamas ?